webpack.config.js

```diff
@@ -0,0 +1,14 @@
+const HtmlWebpackPlugin = require('html-webpack-plugin');
```

Comment:
Did really worked this approach? I thought you had to modify this file https://github.com/solid-studio/solid.web/blob/master/config-overrides.js